datasets/tweet_eval/tweet_eval.py

```diff
@@ -0,0 +1,250 @@
+    },
+    "sentiment": {
+        "train_text": URL + "sentiment/train_text.txt",
+        "train_labels": URL + "sentiment/train_labels.txt",
+        "test_text": URL + "sentiment/test_text.txt",
+        "test_labels": URL + "sentiment/test_labels.txt",
+        "val_text": URL + "sentiment/val_text.txt",
+        "val_labels": URL + "sentiment/val_labels.txt",
+    },
+    "stance": {
+        "abortion": {
+            "train_text": URL + "stance/abortion/train_text.txt",
+            "train_labels": URL + "stance/abortion/train_labels.txt",
+            "test_text": URL + "stance/abortion/test_text.txt",
+            "test_labels": URL + "stance/abortion/test_labels.txt",
+            "val_text": URL + "stance/abortion/val_text.txt",
+            "val_labels": URL + "stance/abortion/val_labels.txt",
+        },
+        "atheism": {
+            "train_text": URL + "stance/atheism/train_text.txt",
+            "train_labels": URL + "stance/atheism/train_labels.txt",
+            "test_text": URL + "stance/atheism/test_text.txt",
+            "test_labels": URL + "stance/atheism/test_labels.txt",
+            "val_text": URL + "stance/atheism/val_text.txt",
+            "val_labels": URL + "stance/atheism/val_labels.txt",
+        },
+        "climate": {
+            "train_text": URL + "stance/climate/train_text.txt",
+            "train_labels": URL + "stance/climate/train_labels.txt",
+            "test_text": URL + "stance/climate/test_text.txt",
+            "test_labels": URL + "stance/climate/test_labels.txt",
+            "val_text": URL + "stance/climate/val_text.txt",
+            "val_labels": URL + "stance/climate/val_labels.txt",
+        },
+        "feminist": {
+            "train_text": URL + "stance/feminist/train_text.txt",
+            "train_labels": URL + "stance/feminist/train_labels.txt",
+            "test_text": URL + "stance/feminist/test_text.txt",
+            "test_labels": URL + "stance/feminist/test_labels.txt",
+            "val_text": URL + "stance/feminist/val_text.txt",
+            "val_labels": URL + "stance/feminist/val_labels.txt",
+        },
+        "hillary": {
+            "train_text": URL + "stance/hillary/train_text.txt",
+            "train_labels": URL + "stance/hillary/train_labels.txt",
+            "test_text": URL + "stance/hillary/test_text.txt",
+            "test_labels": URL + "stance/hillary/test_labels.txt",
+            "val_text": URL + "stance/hillary/val_text.txt",
+            "val_labels": URL + "stance/hillary/val_labels.txt",
+        },
+    },
+}
+
+
+class TweetEvalConfig(datasets.BuilderConfig):
+    def __init__(self, *args, type=None, sub_type=None, **kwargs):
+        super().__init__(
+            *args,
+            name=f"{type}" if type != "stance" else f"{type}_{sub_type}",
+            **kwargs,
+        )
+        self.type = type
+        self.sub_type = sub_type
+
+
+class TweetEvalDataset(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class TweetEval(datasets.GeneratorBasedBuilder):
```
THis must be the camel case version of the dataset script name `tweet_eval`